code/src/components/ThoughtsForm.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './ThoughtsForm.css';
+
+const ThoughtsForm = ({ newThoughtMessage, handleNewThought, onFormSubmit }) => {
+  return (
+    <form className="form-container" onSubmit={onFormSubmit}>
+      <h2 className="question">What&apos;s making you happy right now?</h2>
+      <textarea className="thought-input" value={newThoughtMessage} onChange={handleNewThought} minLength="4" maxLength="140" />
+      <button className="submit-button" type="submit" aria-label="send happy thought" disabled={newThoughtMessage.length < 5 || newThoughtMessage.length > 140}>❤️ Send happy thought ❤️</button>
```

Comment:
Interesting to see a different way of handling the min and max length of characters here to how I did it! Didn’t know you could disable the submit button in this way! I chose to change the colour of the character count instead, like this: 

```
      <div className="characterCount">{eachThought.length < 5 || eachThought.length > 140 ? (
        <p className="redText">{eachThought.length}/140</p>
      ) : (
        <p className="normalText">{eachThought.length}/140</p>
      )}
      </div>
```